database/migrations/2016_04_01_124415_create_customers_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCustomersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('customers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('email')->unique();
+            $table->string('twitter_handle');
+            $table->string('phone_number');
```

Comment:
String is the most appropriate type for phone numbers. we wont do any mathematical operations on them and they wont be used for anything other than editing and adding new ones.